.travis.yml

```diff
@@ -1,3 +1,6 @@
+addons:
+  code_climate:
+    repo_token: 2983486985576ac8d53c4716ab69217e6bad501f42f69138cc796fda85637a6e
```

Comment:
Is this token generated from your GitHub repo fork or a token generated by code climate?